src/server/game/Spells/SpellMgr.cpp

```diff
@@ -3257,6 +3257,20 @@ void SpellMgr::LoadSpellCustomAttr()
                 spellInfo->Effects[EFFECT_0].MiscValue = 127;
                 break;
         }
+
+        if (spellInfo->Speed > 0.0f)
+        {
+            if (SpellVisualEntry const* spellVisual = sSpellVisualStore.LookupEntry(spellInfo->SpellVisual[0]))
+            {
+                if (spellVisual->HasMissile)
```

Comment:
Maybe 1 condition?

Reply:
Do we want to combine where we define spellVisual, maybe it gets messy to read.

But I could do 
```cpp
if (spellVisual->HasMissile && (spellVisual->MissileModel == -4 || spellVisual->MissileModel == -5))
```
If you think that is good?

Reply:
updated